extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+
+![Content Availability](figures/content-availability.jpg)
+
+The purpose of content availability is to check if a file exists before making a network request. If content is marked as unavailable, the network request for that file must be skipped.
+
+A content availability bit must only be set if the corresponding tile availability bit is set. Otherwise, it would be possible to specify content files that are not reachable by the tiles of the tileset. The content availability bitstream can be validated by checking that the following equation holds true:
+
+```
+contentAvailability & ~tileAvailability === 0
+```
+
+where `&` is the bitwise AND operation and `~` is the bitwise NOT operation.
+
+### Child Subtree Availability
+
+**Child subtree availability** is a bitstream that determines what subtrees can be reached from this subtree. There are `N` bits for every node in the bottom-most level of the subtree, where `N` is the branching factor of the tree. A 1 means there is a child subtree available at that position in the tree. Meanwhile, a 0 means there is no subtree available.
+
+![Child Subtree Availability](figures/subtree-availability.jpg)
+
+Child subtree availability is used to determine whether further subtree files exist before making network requests. If a child subtree availability bit is 0, any network request for that subtree must be skipped.
+
+## Subtree JSON Files
+
+A **subtree JSON file** describes where the availability information for a single subtree is stored.
+
+Each subtree JSON file contains the following information:
+
+* The location of a bitstream for tile availability (if not `constant`)
+* The location of a bitstream for content availability (if not `constant`)
+* The location of a bitstream for child subtree availability (if not `constant`)
+
+### Buffers and Buffer Views
+
+OUTLINE:
+- Which spec to reference? 3DTILES_binary_buffers or core metadata?
+
+### Morton Order
+
+>**[Morton order](https://en.wikipedia.org/wiki/Z-order_curve)** assigns indices to nodes in the same level. The indices are used for lookup in availability buffers.
+
+Using the Morton order serves these purposes:
+
+- Efficient tile location decomposition: The Morton order allows efficient encoding and decoding of locations of a tile in the level to its location in the availability buffers.
+- Efficient traversal: The binary representation of tile locations in the tree level allow for easy traversal of the tileset (finding parent and child nodes).
+- Locality of reference: Adjacent indices are stored close to each other in memory and are close to each other spatially.
+
+*The following section is non-normative*
+
+The figure below shows the tile location decomposition of the tile `3/5/1` (Level: 3, X: 5, Y: 1). We first convert the tile location to its Morton index, which is `19`. At Level 3 of a Quadtree, we'll use 6 bits to represent the binary value of the Morton index: `010011`.
```

Comment:
Use tuple notation when referring to coordinates, as template URIs can be configured by the user.

```suggestion
The figure below shows the tile location decomposition of the tile `(level, x, y) = (3, 5, 1)`. We first convert the tile location to its Morton index, which is `19`. At Level 3 of a Quadtree, we'll use 6 bits to represent the binary value of the Morton index: `010011`.
```